deploy.sh

```diff
@@ -152,8 +152,12 @@ svn update
 
 svn status
 
-echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$DEBUG" != false ]]; then
+  echo "➤ Committing files..."
+  svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+else
+  echo "➤ Debug mode: Files not committed."
```

Comment:
This is the only mention so far of the non-live run – how about we provide this feedback as the first output? Of course it would be good to include here, but as someone intending to do a test run, I'd want to know right away if it was in a live or test mode so I could potentially cancel it if needed before it's too late.

Reply:
I like the idea. I have added first output for the script in https://github.com/10up/action-wordpress-plugin-deploy/pull/122/commits/f484e0e8503d198527df7b4c4b614ffc122285e8 with the `ℹ︎ DRY_RUN is true` message.